core/src/components/radio-group/test/form/radio-group.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('radio-group: form', () => {
```

Comment:
Would it make sense to use `toHaveReceivedEventDetail` instead of checking the `#value` text? We could cut out the middleman that way.